README.md

```diff
@@ -3,7 +3,7 @@
 ![logo](https://raw.githubusercontent.com/vthiery/cpp-statsd-client/master/images/logo.svg?sanitize=true)
 
 [![Download](https://api.bintray.com/packages/vthiery/conan-packages/statsdclient%3Avthiery/images/download.svg)](https://bintray.com/vthiery/conan-packages/statsdclient%3Avthiery/_latestVersion)
-[![Build Status](https://travis-ci.org/vthiery/cpp-statsd-client.svg?branch=master)](https://travis-ci.org/vthiery/cpp-statsd-client)
+![Build Status](https://github.com/vthiery/cpp-statsd-client/actions/workflows/linux/badge.svg)
```

Comment:
this will be the badge from the github actions run. as far as i could tell they dont have a link to the latest build though to see like a log and details. seems each "run" gets a hash assigned to it so there is no human readable way to tell what the link should be here

Reply:
Something like `https://github.com/vthiery/cpp-statsd-client/actions?workflow=linux` should work

Reply:
ive done this!